Testing/Data/SystemTest/ILL/D7/394458_394882.nxs.md5

```diff
@@ -0,0 +1 @@
+b68d0c03e7f26bc8441a7f2200c5c0b3
```

Comment:
No need to duplicate these in system tests, system test jobs see the unit test directory as well.

Reply:
OK, that didn't work locally at first so I added them in both places. Will be removed then.